packaging/linux/keybase.desktop

```diff
@@ -6,4 +6,4 @@ Terminal=false
 Type=Application
 Categories=Network;
 StartupNotify=false
-MimeType=x-scheme-handler/web+stellar;
+MimeType=x-scheme-handler/web+stellar;x-scheme-handler/keybase;
```

Comment:
what is the "web+" for in stellar://? do we need it?
https://developer.mozilla.org/en-US/docs/Web/API/Navigator/registerProtocolHandler
>For security reasons, registerProtocolHandler() has restrictions on which schemes may be registered. A custom scheme may be registered as long as the scheme's name begins with "web+", is at least five characters long (including the "web+" prefix), and has only lower-case ASCII letters in its name. For example, "web+burger", as shown in the Example below.

Reply:
@heronhaye that's a web browser API -- we're registering a scheme handler outside a web browser, so it can be whichever scheme we like.

`web+stellar:` was chosen by the Stellar folks, I guess they wanted it to be registerable by web pages too?  We don't want `keybase://` to be registerable by web pages, so I think we're good as-is.

Reply:
@heronhaye those are sep7 links https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0007.md

Reply:
oh interesting. sounds good then. i think you have to add a case in run_keybase, it only accepts web+ links right now

Reply:
@heronhaye Thanks!  How's 8315da4ba9?  I noticed we had an unescaped `+`, too.

Reply:
nice catch. seems ^(web\+|keybase).* needs to be in single quotes too (`'^(web\+|keybase).*'`). so tricky...

Reply:
@heronhaye single quotes seems to break it for me

Reply:
my bad, I was testing on zsh instead of bash. lgtm